app/views/wiki/_header.html.erb

```diff
@@ -31,7 +31,13 @@
 
     <div style="padding-top:8px;text-align:center">
       <span class="d-none d-lg-inline">
-        <span rel="tooltip" title="The number of authors for this page." data-placement="top"><i style="color:#888;" class="fa fa-user"></i> <%= number_with_delimiter(@node.authors.size) %></span>  |
+        <span rel="popover" style='cursor: pointer;' title="Authors" data-html="true" data-placement="top" data-content= "
+        <ul>
+        <% @node.authors.each do |author| %>
+          <li style='list-style: none;'><a href='/profile/<%= author.username %>' style='color: #009bff;'>@<%= author.username %></a></li>
+        <% end %>
+      </ul>
+      "> <i style="color:#888;" class="fa fa-user"></i> <%= number_with_delimiter(@node.authors.size) %></span>  | 
```

Comment:
@Priyaraj17, this is the file @Tlazypanda was mentioning. These changes are not required.